src/sdgio-edit.owl

```diff
@@ -3187,13 +3201,13 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
 AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020286> "improved forecasting of impacts of climate change and targeted technology development"@en)
 ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020237> <http://purl.unep.org/sdg/SDGIO_00020286>)
 
-# Individual: <http://purl.unep.org/sdg/SDGIO_00020287> (enhanced capacity to deal with climactic risks and extremes through the achievement of mitigation and adaptation plans)
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020287> (enhanced capacity to deal with climatic risks and extremes through the achievement of mitigation and adaptation plans)
```

Comment:
This is still phrased like an outcome rather than an objective.
It should be "enhance capacity to ..."

Reply:
yes, it is one of the many sub-IDOs that we inserted a couple of weeks ago the labels and definitions of which are not objective-sounding. I fix this here, but more needs attention. I rewrote the labels in the google sheet Process classes for DOs in the MARLO folder.

Reply:
Make a dedicated issue. This is my fault, I should have seen this earlier.

Reply:
I created issue #166 for this.

Reply:
Mostly resolved #166 with https://github.com/SDG-InterfaceOntology/sdgio/pull/162/commits/c804d54c4ad06377e5907c0be8b59745339b0181

Reply:
Yes, there are still items the items of which I need to change. I have not done it yet because we decided to open a new PR for those later. I suppose that I should not close issue #166 yet.

Reply:
Not yet, but I attended to most of it